### DISCUSSION TITLE
Add support for multi-task learning

### DISCUSSION THREAD

**Discussion by xyk2000** [autogluon/autogluon#1724] [Q&A] [answered]
Automated machine learning for multimodal multitasking would be interesting

**Comment by xyk2000** [comment]
I recently wanted to implement a multi-objective regression task and was keen to simplify the training effort with autogluon. Is it foreseeable to add support for multitasking to the existing code base?

**Comment by Innixma** [comment, marked as answer]
Refer to the multi-label tutorial for an example of this: https://auto.gluon.ai/stable/tutorials/tabular_prediction/tabular-multilabel.html
